Source/VSProps/Base.props

```diff
@@ -113,8 +113,9 @@
       4121  'symbol' : alignment of a member was sensitive to packing
       4324  Padding was added at the end of a structure because you specified a __declspec(align) value.
       4714  function 'function' marked as __forceinline not inlined
+      5054  operator '+': deprecated between enumerations of different types
```

Comment:
Should this warning *really* be disabled by default rather than being resolved? This is likely going to bite us down the road when it actually gets removed from the language.

Reply:
From what I remember, it was mainly in the Qt headers. I suppose we could just disable it in the `DolphinQt` project instead?

Reply:
Yeah, that sounds fine. At least that way, Core and other projects can still let the compiler complain loudly when it happens.

Reply:
There have been PR's pre-emptively fixing this in main code, so yes I also think disabling it only in Qt makes sense. Moreover, I think it's time we purge some suppressed warnings, I once started it but don't think I went far.